website/blog/2021-02-04-64updates.md

```diff
@@ -0,0 +1,98 @@
+---
+title: "Gallery App, TurboModules, and more community modules"
+author: Kiki Saintonge, Harini Kannan, Steven Moyes
+authorURL: https://twitter.com/ReactNativeMSFT
+description: "More modules and tools released in React Native 0.64"
+is_blog: true
+published_date: February 4, 2021
+---
+
+React Native 0.64 is out! Which means it's time to showcase all the work we've put in around supporting more community modules, TurboModules, APIs improvements, and even some tools, all in the effort to improve the developer experience.

```

Comment:
Re TurboModules, last I heard from @acoates-ms was that we wanted to not yet document it.

@vmoroz  can you confirm?

Reply:
@vmoroz, @acoates-ms or @NickGerleman did we find out whether TurboModules is something we should highlight this release?

Reply:
I'd move opt in hermes up to the top level (title / intro), and drop TurboModules from the same places.  But keep the turbo module topic in the contents.